ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -276,7 +276,7 @@ extern "C" {
     pub fn Servo_StyleSheetHasRules(sheet: *mut RawServoStyleSheet) -> bool;
     pub fn Servo_InitStyleSet() -> *mut RawServoStyleSet;
     pub fn Servo_DropStyleSet(set: *mut RawServoStyleSet);
-    pub fn Servo_ParseStyleAttribute(bytes: *const u8, length: u32,
```

Comment:
This reverts https://github.com/servo/servo/commit/0f09a9f357c3c038ffc0b138772626f2d4256837. Is that intended?